class-coblocks.php

```diff
@@ -115,17 +116,7 @@ private function includes() {
 			require_once COBLOCKS_PLUGIN_DIR . 'includes/ical-parser/class-coblocks-event.php';
 			require_once COBLOCKS_PLUGIN_DIR . 'includes/ical-parser/class-coblocks-ical.php';
 
-			// Require the Gutenberg plugin for specific components.
-			include_once ABSPATH . 'wp-admin/includes/plugin.php';
-			$installed_plugins = get_plugins();
-
-			$gutenberg_plugin_file    = 'gutenberg/gutenberg.php';
-			$gutenberg_plugin_version = empty( $installed_plugins[ $gutenberg_plugin_file ] ) ? null : $installed_plugins[ $gutenberg_plugin_file ]['Version'];
-
-			// Require CoBlocks custom filters
-			require_once COBLOCKS_PLUGIN_DIR . 'src/components/gutter-control/gutter-wrapper.php';
```

Comment:
@jrtashjian did that move somewhere else?

Reply:
Mistakenly removed. Fixed in 3aee61c